src/Serialization/containers.jl

```diff
@@ -55,17 +59,27 @@ end
 
 ################################################################################
 # Saving and loading matrices
+
+encodeType(::Type{<:Matrix}) = "Matrix"
+reverseTypeMap["Matrix"] = Matrix
+
 function save_internal(s::SerializerState, mat::Matrix{T}) where T
     m, n = size(mat)
     return Dict(
         :matrix => [save_type_dispatch(s, [mat[i, j] for j in 1:n]) for i in 1:m]
     )
 end
 
+# deserialize vector with specific content type
```

Comment:
```suggestion
# deserialize matrix with specific content type
```
Or if you want to keep the comments identical everywhere, say "object" or "container" instead of "vector"?